lib/resources/r4/document_reference.yaml

```diff
@@ -290,3 +291,42 @@ fields:
         "end": "2020-01-01T01:00:00.000Z"
       }
     }
+
+- name: docStatus
+  required: 'Yes'
+  type: code
+  action:
+    - update
+  description: The status of the underlying document.
+  note: Only `final` is supported for provider access. `final` and `amended` are supported for system access.
```

Comment:
We'll support both final and amended for all access types for updates.  We're also allowing 'entered-in-error' as a way to in-error documents.

Reply:
You can find the changes here: https://github.com/cerner/fhir.cerner.com/pull/463/commits/2697a167ece46ac0a9e9b0a2a671208d4ce9715d